### ISSUE TITLE
Lazygit from neovim inside tmux is slightly different than vanilla lazygit

### ISSUE THREAD

**Issue by shauryagoel** [jesseduffield/lazygit#3829]
**Describe the bug**
If I open lazygit from neovim which is inside tmux I get different pane sizes than when opening lazygit from terminal inside tmux without neovim or when opening lazygit from neovim without tmux or when opening lazygit from terminal without tmux. 
Lazygit from neovim inside tmux- 
<img width="664" alt="image" src="https://github.com/user-attachments/assets/d8870823-bc63-4bd7-868b-d984b4d379f6">

Lazygit inside neovim and without tmux- 
<img width="670" alt="image" src="https://github.com/user-attachments/assets/409d9b8e-ced2-457b-b891-41a6a50b4bce">

As you can see, the pane sizes on the left-hand side are different in both cases.

**To Reproduce**
Steps to reproduce the behaviour:

1. Open a new tmux session
2. Open neovim inside it (`nvim --clean`)
3. Open terminal inside neovim (`:term`)
4. Type lazygit and press enter

For the second screenshot skip step 1 and follow from step 2 onwards.

**Expected behaviour**
In both cases, the corresponding pane sizes should be the same.

**Screenshots**
Added above while describing the bug.

**Version info:**
lazygit version- `commit=58d746718056ea7ca802aafdd244d6abeb7255f8, build date=2024-08-09T10:19:46Z, build source=unknown, version=58d74671, os=darwin, arch=arm64, git version=2.44.0`
git version- `git version 2.44.0`
neovim version- `v0.10.1`
tmux version- `tmux 3.3a`
OS: `MacOS 14.4`

**Additional context**
1. I have manually built the lazygit using the commit 58d74671.
2. I get the same result even after disabling any config for tmux and neovim.

If you want to try and debug this issue yourself, you can run `lazygit --debug` in one terminal panel and `lazygit --logs` in another to view the logs.
--> This does not show any useful errors

**Comment by shauryagoel** [comment]
Yes you are right. I decreased the font size and now everything is working as expected. Is there a way to configure this threshold somewhere? I don't prefer shrinking of non-focused side panels.

**Comment by shauryagoel** [comment]
Hi @stk-ableton, is there some way to config? I am unable to find any such config option.

**Comment by stefanhaller** [comment]
No, it's hard-coded.

**Comment by ChrisMcD1** [comment]
/close

Closing as part of a cleanup of old issues, seems this one is resolved (albeit in an unsatisfactory way)
